exporter/tanzuobservabilityexporter/metrics.go

```diff
@@ -51,10 +51,10 @@ var (
 
 var (
 	// Specifies regular histogram
```

Comment:
This comment and the one below seem redundant given the name of the vars and structs.

Reply:
Done. Thanks for the feedback.